app/components/onboarding/AddNewSite.tsx

```diff
@@ -12,47 +14,99 @@ import {
 } from '@mui/material';
 import { palettes } from '@/app/[lang]/theme/palettes';
 import { Add, ArrowBack, PlaceOutlined } from '@mui/icons-material';
-import AddNewSiteDialog from '@/app/components/onboarding/AddNewSiteDialog';
 import TitleBar from '@/app/components/onboarding/TitleBar';
 import type {
   SiteData,
   UserFormData,
 } from '@/app/components/onboarding/OnboardingDashboard';
 import type { Dict } from '@/app/[lang]/dictionaries';
 import { cropTypes } from '@/app/[lang]/dictionaries';
+import { LatLng, LatLngExpression } from 'leaflet';
+import dynamic from 'next/dynamic';
+
+const AddNewSitePosition = dynamic(
```

Comment:
Is this necessary? I tried importing it the regular way and had no errors so far